### ISSUE TITLE
The stacking order problem

### ISSUE THREAD

**Issue by gregoriopellegrino** [w3c/epub-specs#2910]
### Section

The stacking order problem

### Describe the problem

The issue is strictly related to EPUB authoring software and is not generic to the standard.

### Describe the fix or new feature you propose

I suggest removing it.

**Comment by clapierre** [comment]
From the A11Y TF discussion.
Gregorio Pellegrino: sometimes you can't control the reading order and was the order when you add element to the page for authoring tools.
… may give guidance using css for visual order with z index

Avneesh Singh: stacking order problem issue with some production tools. we need to make the decision at the principal level.

RESOLUTION: We should have production/authoring tools independent techniques in fixed layout techniques document, if there is a strong demand from community to have production tool specific guidance, then we can consider a separate document.
